tests/unit/core/test_utils.js

```diff
@@ -1130,3 +1131,27 @@ describe('parsePage', () => {
     expect(parsePage(undefined)).toBe(1);
   });
 });
+
+describe('sanitizeUserHTML', () => {
```

Comment:
I think this could use another test for supplying a `null` value that isn't a string. We probably don't want it... but we should either treat it like an empty string or throw an error. I just don't know what will happen looking at the code, so I think we should have a test to define the expected behaviour.

Reply:
Good call, I added it.